frontend/src/api/djangoAPI.js

```diff
@@ -1,5 +1,59 @@
 import axios from 'axios';
 
-export default axios.create({
+/**
+ * initialize the client with the base url
+ */
+const http = axios.create({
     baseURL: process.env.REACT_APP_DJANGO_API
-})
\ No newline at end of file
+})
+
+class djangoAPI {
+
+    /**
+     * performs GET request to retrieve full videos list
+     *
+     * @returns {Promise<void>}
+     *          videos list
+     */
+    getAllVideos() {
```

Comment:
I think we don't really need that method.
It can be clear enough that when we pass no search query as argument on GET /videos, all the videos are retrieved.
A little comment in the docstring of the next method would be enough.